Activity feed has more filters and a sort option

AlexHalb opened the pull request:
Resolves #100 

Note that I added the sort option by reversing the results locally if the 'oldest first' option is selected. This is definitely not optimal, but it would solve some issues with finding things and there doesn't seem to be a backend option.

luxaritas commented:
Doing oldest first client side will cause more issues than the slim benefit that wold provide - that needs to be done server side. The reward notification still needs to be implemented too, right?

AlexHalb commented:
I've added reward notifications and removed the reverse sort filter. The wording of the notifications might need to be tweaked - I don't have an example from the API to work from and had to make a couple educated guesses.

luxaritas commented:
Just crosschecked with my account on legacy - texts look fine
